discordbot/economy/economy_menu.py

```diff
@@ -91,22 +94,40 @@ def check(reaction, user):
                 "reaction_add", timeout=cfg.MENU_TIMEOUT, check=check
             )
             if reaction.emoji == "0️⃣":
-                await overview_command(ctx)
+                if cfg.DEBUG:
+                    print("Reaction selected: 0")
+                await overview_command(ctx, "")
```

Comment:
I think the argument should be handled from the other side.

I.e. we should have a default on overview_command, that is like this:
```
overview_command(ctx, arg="")
```

This is because imagine that we absolutely need one argument to have the functionality working. The controller shouldn't have to select this. It should be responsibility of the function implementation to have a saying over it.

Reply:
Also, since we are doing the print in the controller, I think we should do the other commands prints in the controller as well. So inside each function, we basically just do a print if there's a try-catch error.

Reply:
Oh yeah you're right it makes the code way more robust. Will fix this